src/fluree/db/dbfunctions/internal.cljc

```diff
@@ -640,13 +640,14 @@
       sum)
     (catch* e (function-error e "objF" flakes))))
 
-;; TODO - below is java-specific, need a javascript version
 (defn rand
   [instant max']
   (try*
-    (let [base (.nextDouble (java.util.Random. instant))
-          num  (int (Math/floor (* base max')))] num)
-    (catch* e (function-error e "rand" instant max'))))
+    (let [rng #?(:clj (java.util.Random. instant) :cljs (seedrandom instant))
```

Comment:
The cljs compiler is warning that seedrandom is undeclared and that it can't infer the type of `base` here. Do we need to add it as an npm dep?

Reply:
^ @cap10morgan

Reply:
Yes, sorry. Coming shortly.

Reply:
Node REPL tested!

Reply:
Thanks!